subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/ReportFileProvider.kt

```diff
@@ -0,0 +1,28 @@
+package com.avito.android.test.report.transport
+
+import com.avito.android.Result
+import java.io.File
+
+interface ReportFileProvider {
+
+    val rootDir: File
+
+    /**
+     * special string, that indicates that file should be uploaded
+     * and string should be changed with url later by test runner
+     */
+    fun toUploadPlaceholder(file: File): String
+
+    /**
+     * @return filename with extension
+     */
+    fun fromUploadPlaceholder(placeholder: String): String
```

Comment:
Can we make a symmetrical API?

- toUploadPlaceholder(file): String
- fromUploadPlaceholder(path): File

Reply:
Looks like no, 
we don't carry full path, only filename